### PR TITLE
Add parameter to translate site_name

### LINKED ISSUE #80: Translate site_name
Is there a way to translate the `site_name` parameter ?

A plugin parameter could be great : 

```yaml
site_name: Mon super site
plugins:
  - i18n:
      default_language: fr
      languages:
        fr: Français
        en: English
      site_name_translations:
        en: My awesome website
```

### PR DESCRIPTION
**Pull request by brospars**
Closes #80

### CODE REVIEW

File: mkdocs_static_i18n/plugin.py [hunk 1]
```diff
@@ -422,6 +423,9 @@ def on_page_context(self, context, page, config, nav):
                 alternate["link"] += f"{separator}{page_url}"
             config["extra"]["alternate"] = alternates
 
+        if self.config["site_name_translations"].get(page.file.dest_language, {}):
```
Comment (ultrabug) [inline mkdocs_static_i18n/plugin.py:426]:
```suggestion
        if self.config["site_name_translations"].get(page.file.dest_language):
```

I'm not sure adding a default of `{}` is useful on a if block like this one

**Comment by ultrabug** [comment]
Thanks @brospars 

- please check my comment above
- please make sure CI passes, I guess you missed to run black etc (`make qa` should help you)
- finally, could you update the `README` to document this new option with an example please?

I also think that tests should be added, but if you cba with it, I'll do it later

**Comment by brospars** [comment]
Hello,
Thanks for your review. I'll do my best to fix the CI and add a test case.
Maybe next week.

**Comment by brospars** [comment]
I've rebase and made the changes your required. I'll add a test case later.

**Comment by ultrabug** [comment]
@brospars you're gonna hate me but I think that since I'm [working on implementing your idea](https://github.com/ultrabug/mkdocs-static-i18n/issues/5#issuecomment-1027958011) we actually do not need a new option!

Instead, we could add a `site_name` key to languages right? :)

**Comment by brospars** [comment]
You're right 👍 so you should probably close this PR 😉

**Pull request closed without merging**